src/main/scala/com/exasol/cloudetl/kinesis/KinesisUserProperties.scala

```diff
@@ -0,0 +1,83 @@
+package com.exasol.cloudetl.kinesis
+
+import scala.collection.SortedMap
+
+import com.exasol.cloudetl.common.{AbstractProperties, CommonProperties}
+
+/**
+ * A companion object for [[KinesisUserProperties]] class.
+ */
+object KinesisUserProperties extends CommonProperties {
+  val AWS_ACCESS_KEY_PROPERTY: String = "AWS_ACCESS_KEY"
+  val AWS_SECRET_KEY_PROPERTY: String = "AWS_SECRET_KEY"
+  val AWS_SESSION_TOKEN_PROPERTY: String = "AWS_SESSION_TOKEN"
+  val STREAM_NAME_PROPERTY: String = "STREAM_NAME"
+  val REGION_PROPERTY: String = "REGION"
+  val TABLE_NAME_PROPERTY: String = "TABLE_NAME"
+
+  /**
+   * Creates an instance of [[KinesisUserProperties]] from user provided key value
+   * properties.
+   */
+  def apply(params: Map[String, String]): KinesisUserProperties =
+    new KinesisUserProperties(params)
+
+  /**
+   * Creates an instance of [[KinesisUserProperties]] from a user provided formatted
+   * string with properties.
+   */
+  def apply(string: String): KinesisUserProperties =
+    apply(mapFromString(string))
+}
+
+/**
+ * This class parses and stores Kinesis user's properties.
+ */
+class KinesisUserProperties(val propertiesMap: Map[String, String])
+    extends AbstractProperties(propertiesMap) {
+  import KinesisUserProperties._
+
+  final def getAwsAccessKey: String =
+    getString(AWS_ACCESS_KEY_PROPERTY)
+
+  final def getAwsSecretKey: String =
+    getString(AWS_SECRET_KEY_PROPERTY)
+
+  final def getAwsSessionToken: String =
+    getString(AWS_SESSION_TOKEN_PROPERTY)
+
+  final def getStreamName: String =
+    getString(STREAM_NAME_PROPERTY)
+
+  final def getRegion: String =
+    getString(REGION_PROPERTY)
+
+  final def getTableName: String =
+    getString(TABLE_NAME_PROPERTY)
+
+  /**
+   * Converts a properties map to a string.
+   *
+   * @param propertiesMap A map with properties.
+   */
+  final def makeString(propertiesMap: Map[String, String]): String =
```

Comment:
Please use `mkString` method. 
It is general convention for Scala [structures](https://www.scala-lang.org/api/2.12.3/scala/collection/immutable/List.html#mkString(sep:String):String).

Reply:
done